### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by mattwthompson**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,14 +29,14 @@ requirements:
...
     - openff-qcsubmit
-    - forcebalance >=1.9.2
-    - openff-fragmenter-base >=0.1.2
-    - openmm >=7.6.0
+    - forcebalance
+    - openff-fragmenter-base
+    - openmm
     - qcportal >=0.15.6
-    - qcengine >=0.23.0
+    - qcengine >=0.25
```
Comment (mattwthompson) [inline recipe/meta.yaml:39]: The toolkit constraint is not pushed to their packaging, but this is the version that should be compatible

https://github.com/MolSSI/QCEngine/blob/master/docs/source/changelog.rst#v0250--2022-11-11

**Review by j-wags — APPROVED** [review]
Looks good to me! Glad to see the version restrictions go.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -29,14 +29,14 @@ requirements:
...
-    - openff-toolkit-base >=0.10.0,<0.11.0a0
+    - openff-toolkit-base >=0.11
     - openff-forcefields
     - openff-qcsubmit
-    - forcebalance >=1.9.2
-    - openff-fragmenter-base >=0.1.2
-    - openmm >=7.6.0
+    - forcebalance
+    - openff-fragmenter-base
+    - openmm
```
Comment (j-wags) [inline recipe/meta.yaml:37]: 👍

**Pull request merged**